### DISCUSSION TITLE
Template post upgrade task to include list of dependencies upgraded

### DISCUSSION THREAD

**Discussion by RyanFrench** [renovatebot/renovate#10987] [Request Help]
We have an internal tool that is used to automatically update our CHANGELOG as part of our PR process. We are running Renovate on-prem with a GitHub Enterprise installation.

In our docker container, we have the following in the `config.json`

```json
{
  "allowPostUpgradeCommandTemplating": true,
  "allowedPostUpgradeCommands": [
    "^release bump patch changed"
  ]
}
```

and in our shared configuration file we have

```json
{
  "major": {
    "enabled": false
  },
  "ignorePaths": [
    "**/node_modules/**",
    "**/bower_components/**",
    "**/vendor/**",
    "**/examples/**",
    "**/__tests__/**",
    "**/test/**",
    "**/tests/**",
    "**/__fixtures__/**"
  ],
  "branchPrefix": "renovatebot/",
  "groupName": "all",
  "prCreation": "immediate",
  "postUpdateOptions": ["gomodTidy", "npmDedupe"],
  "postUpgradeTasks": {
    "commands": [
      "release bump patch changed 'Renovate[bot] - Upgraded dependencies{{#each upgrades}}\n&nbsp;&nbsp;&nbsp;&nbsp;* Upgraded {{this.depName}}: {{this.currentVersion}} => {{this.newVersion}}{{/each}}'"
    ],
    "fileFilters": ["**/**"],
    "executionMode": "branch"
  }
}
```

What I'm aiming to do is list the dependencies that have been upgraded with their versions in the changelog by passing them into the post upgrade task, but all I'm seeing in the CHANGELOG is

```
Renovate[bot] - Upgraded dependencies
```

**Comment by latipun7** [comment]
Same problem here.
`renovate-on-prem` v1.5.0

Global config already has `allowPostUpgradeCommandTemplating` set to `true`.

Repo config has this command:
```json
"commands": ["yarn set version {{{newVersion}}}"]
```

But it gives artifact error in my case, the error is `Command failed: yarn set version  `, the `newVersion` variable translated as empty string. I even tried with `newValue` too, but the template still give empty strings.

Any insight? Thank you.
